source/_integrations/tuya.markdown

```diff
@@ -25,79 +24,123 @@ ha_platforms:
 
-## Integration Options
+#### Link devices by app account
 
-It is possible to change various behaviors through the integration options, some common for integration and others specific to each `light` and `climate` devices. These can be changed at **Tuya** -> **Options** on the Integrations page.
+Link devices by your app account and copy the **Device ID** in the **Device List** to the clipboard for specifying `device_id` later.
 
-### Common Options
+**1.** Navigate to the **Devices** tab.
 
-- **Discovery device polling interval** (default=605): define the interval between 2 consecutive calls to the `API discovery method`, which is used to get the status for all registered devices with a single call. If you set interval value too low, Tuya API will return errors, so it is suggested to use the default value until
-you know that it is possible to use lower values.
+**2.** Click **Link Tuya App Account** > **Add App Account**.
+![](../images/integrations/tuya/image_004.png)
 
-- **Query device polling interval** (default=120): this option is available only if you have devices that can use the `API query method`. 
-It defines the interval between 2 consecutive calls to the `API query method`, that is used to get the status for a specific device. 
-This method is always used when it is available only one device that can use it. If you set interval value too low, Tuya API will return errors 
-so it is suggested to use the default value until you know that is possible to use lower values.
+**3.** Scan the QR code that appears using the **Tuya Smart** app or **Smart Life** app.
+![](../images/integrations/tuya/image_005.png)
 
-- **Device that will use the query method**: this option is available only if you have devices that can use the `API query method`. 
-Because it is not possible to make multiple calls to the `API query method`. If you have more than one device that can use it you can choose which one will use. This will give a better status refresh for this specific device.
+**4.** Click **Confirm** on your **Tuya Smart** or **Smart Life** app.
 
-- **Device to configure (multi-select list)**: this option is available only if you have a `light` or `climate` device. Selecting a device to 
-configure to options page related to the device will be opened. You can also select more than one devices to configure them simultaneously, 
-but all selected devices must be of the same type.
+**5.** Navigate to the **Device List** tab. You can find the devices that have been added through your **Tuya Smart** or **Smart Life** account.   
+![](../images/integrations/tuya/image_006.png)
 
-### Light Options
+</details>
 
-- **Force color support**: when checked force `color support` for devices that do not report this feature.
+### Install the Tuya Home Assistant integration
 
-- **Brightness range**: change the `brightness range` used for the device. Possible options are:
-    - range 1-255 (default)
-    - range 10-1000
+#### Install Home Assistant
```

Comment:
This is not the Home Assistant installation manual and doesn't need to be part of our documentation. Please remove this section.